package.json

```diff
@@ -8,17 +8,20 @@
     "analyze": "yarn workspace patternfly-org analyze",
     "develop": "yarn develop:v5",
     "develop:v5": "yarn workspace patternfly-org develop",
+    "develop:extensions": "EXTENSIONS_ONLY=true PRERELEASE=true yarn develop:v5",
     "build": "yarn build:v5",
     "build:analyze": "yarn workspace patternfly-org build:analyze && yarn copy:v5",
     "build:v4": "yarn build:v5",
     "build:v5": "yarn workspace patternfly-org build && yarn copy:v5",
+    "build:extensions": "EXTENSIONS_ONLY=true PRERELEASE=true yarn build:v5",
```

Comment:
EXTENSIONS_ONLY env variable indicates that this is an extensions build.  PRERELEASE env variable indicates that you would like to include sections / docs that are not ready for release yet in this build (e.g. Component Groups section).